frontend/src/utilities/utils.ts

```diff
@@ -183,3 +183,9 @@ export const isEnumMember = <T extends object>(
   }
   return false;
 };
+
+export const isInternalRouteIntegrationsApp = (internalRoute?: string): internalRoute is string =>
+  internalRoute?.startsWith('/api/') ?? false;
+
+export const isIntegrationApp = (app: OdhApplication): boolean =>
```

Comment:
After you move the type to the types file from the previous comment. Could you update this to help connect the two?

```suggestion
export const isIntegrationApp = (app: OdhApplication): app is OdhIntegrationApplication =>
```

Reply:
Done